aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -948,12 +973,18 @@ static void addmm_impl_cpu_(
             beta.to<scalar_t>(),
             c.data_ptr<scalar_t>(), ldc);
       });
-
   if (!c.is_same(result)) {
     result.copy_(c);
   }
 }
 
+
+TORCH_IMPL_FUNC(addmm_cpu_out)(const Tensor &self, const Tensor &m1, const Tensor &m2, const Scalar& beta, 
```

Comment:
TODO: Use `const_cast<Tensor&>` wherever necessary instead of using this function as a wrapper.

Reply:
Can't be done, as `std::swap` is being used.